### DISCUSSION TITLE
List of available short-cut keys

### DISCUSSION THREAD

**Discussion by oxytoc** [sourcelair/xterm.js#5025] [Q&A]
I can't find a list of all available xterm.js shortcuts anywhere. Please give me a link if there is one, if not, could you write them all down with a short description?

**Comment by jerch** [comment]
What do you mean by "xterm.js shortcuts"? API docs? VT sequence docs? You will find both on https://xtermjs.org. VT sequences are a bit tricky to understand at first, you may want to check https://en.wikipedia.org/wiki/ANSI_escape_code for an introductory primer.
